scripts/ontoportal.js

```diff
@@ -488,4 +489,12 @@ jQuery(document).ready(function ($) {
 
         return $result;
     }
+
+    // Focus on input text field when open select2 item
+    if (focusOnOpen) {
+        // see https://forums.select2.org/t/search-being-unfocused/1203/14
+        $(document).on("select2:open", () => {
+            document.querySelector(".select2-container--open .select2-search__field").focus();
+        });
+    }
```

Comment:
Realize focus only in the select block "ontoportalAddSelect".
No need for a focusOnOpen condition in this case.